docs/exploration-and-organization/exploration.md

```diff
@@ -98,17 +84,51 @@ Some things to remember with bookmarks:
 - Items that you bookmark will get a boost in your search results (but not the search results of other people).
 - To reorder bookmarks, simply drag and drop them in the sidebar.
 
-## Verified items
+## Search
```

Comment:
I agree with you, we can have a separate page about "Search", also might be good idea for a small video.